### PR TITLE
Css page result

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by Elfaram — DISMISSED** [review]

**Review by Elfaram — DISMISSED** [review]

File: frontend/src/components/FilmDetails/index.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          className="banner"
+          style={{
+            backgroundImage: `url(https://image.tmdb.org/t/p/original${moviesData.backdrop_path})`,
+          }}
+        >
+          <div className="info">
+            <img
+              className="img"
+              src={`https://image.tmdb.org/t/p/w300${moviesData.poster_path}`}
+              alt=""
```
Comment (LuigiMenez) [inline frontend/src/components/FilmDetails/index.jsx]: missing alt, useful for accessibility

File: frontend/src/components/CarrousselNews/index.jsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ function CarrousselNews() {
           {moviesData.map((movie) => (
             <img
               key={movie.id}
-              src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
+              src={`https://image.tmdb.org/t/p/original${movie.backdrop_path}`}
               alt=""
```
Comment (LuigiMenez) [inline frontend/src/components/CarrousselNews/index.jsx]: missing alt, useful for accessibility

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
missing alt, useful for accessibility

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Review by Elfaram — DISMISSED** [review]

**Review by NtGZeko — COMMENTED** [review]
change alt

**Review by Elfaram — DISMISSED** [review]

**Review by Elfaram — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Review by Elfaram — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**